backend/src/main/resources/application-dev.properties

```diff
@@ -18,11 +18,17 @@ thread.name.prefix=default_executor_thread
 
 # Local MySql Instance Configuration
 spring.cloud.gcp.sql.enabled=false
-spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:mysql://localhost:3306/rolecall_db?serverTimezone=UTC
 spring.datasource.username=rolecall
 spring.datasource.password=1234
 
+## Hibernate Properties
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+# originally no dialect
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
```

Comment:
@zephyr-l  please check this

Reply:
I removed some extra statements that helped when my local db was MySQL 8.0.

I believe the only remaining difference is the order of statements and some comments.

Reply:
Cloud SQL uses instance running MySQL 5.7. Based on my understanding this is optimizing the query and might cause backward compatible issues. I would suggest to remove this particular line in checked code and use only for local testing.